Module @meetfranz/theme/lib/index.js cannot be found on launch

<!--
Please remember that if you are logging a bug for some service that has *stopped working*, please log the bug at https://github.com/getferdi/recipes/issue
Please remember to read the self-help documentation at https://github.com/getferdi/ferdi#troubleshooting-recipes-self-help - in case it helps you unblock yourself.

Please fill our form below, this way we can analyse and fix the problem as fast as possible.

Please keep in mind that any text inside "<!--" and "--\>" are comments from us and won't be visible in your bug report, so please don't put any text in them.
-->
**Bug description:**
<!-- Add a clear and concise description of what the bug is -->
Ferdi doest not open the GUI. I am using Ubuntu 20.04. Using a .desktop shortcut in Linux I get no feedback at all. With a process monitor I can see that the process keeps running but no GUI opens. On further investigating I started Ferdi via shell and got the traceback attached in the debug section of this issue

**Steps to reproduce:**
<!-- Explain to us how we can reproduce your bug on our computer -->
I was only able to test it on Ubuntu 20.04.
1. Build the current code (in my case with docker) or use the gfolder nightly [Ferdi-amd64-1261c4aa](https://drive.google.com/drive/folders/1vAk84hvZ5ftN6nWv4vv2a088efqtk7av)
2. Install
3. Try to launch
4. When launched via shell, you should see the error

**Expected behavior**
<!-- If applicable, add a description of what you expected to happen. -->
Ferdi starts correctly


**Environment:**

- Operating System: Ubuntu 20.04
- Ferdi Version: 5.6.0-nightly.18
- Server: Using without an account 

- Debug information:
Crash after running `/opt/Ferdi/ferdi`
```bash
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/opt/Ferdi/resources/app.asar/node_modules/@meetfranz/theme/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:310:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:523:18)
    at Module._resolveFilename (internal/modules/cjs/loader.js:874:27)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:261:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/opt/Ferdi/resources/app.asar/config.js:1:1307)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
```

--- a/.dockerignore
+++ b/.dockerignore
@@ -19,9 +19,6 @@ appveyor.yml
 Dockerfile
 flow-typed
 out
-packages/forms/lib
-packages/theme/lib
-packages/ui/lib
 recipes/archives
 tmp-out/
 uidev/lib

